<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointSurveyDialogBase</class>
 <widget class="QDialog" name="PointSurveyDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Survey</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="mStrataLayerLabel">
       <property name="text">
        <string>Strata layer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="mMinimumDistanceAttributeComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mSurveyAreaLabel">
       <property name="text">
        <string>Survey area layer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mSurveyAreaLayerComboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mStrataLayerComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mMinDistanceLabel">
       <property name="text">
        <string>Minimum distance attribute</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="mNSamplePointsComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mNSamplePointsAttribute">
       <property name="text">
        <string>N sample points attribute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>296</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="mCreateSampleButton">
     <property name="text">
      <string>Create sample sites</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PointSurveyDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PointSurveyDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
